src/commands/deploy/deploy.ts

```diff
@@ -66,9 +66,9 @@ export async function deploy(actionContext: IActionContext, arg1?: vscode.Uri |
             const linuxFxVersion: string = siteConfig.linuxFxVersion.toLowerCase();
             if (linuxFxVersion.startsWith(LinuxRuntimes.node)) {
                 // if it is node or python, prompt the user (as we can break them)
-                await promptScmDoBuildDeploy(context, context.effectiveDeployFsPath, LinuxRuntimes.node);
+                await enableScmDoBuildDuringDeploy(context.effectiveDeployFsPath, LinuxRuntimes.node);
```

Comment:
Removed the prompt for SCM_DO_BUILD_DEPLOY. SCM_DO_BUILD_DEPLOY will be enabled by default when an app is deployed.

Reply:
I'd like this to also set an app setting rather than create the .deployment file with the setting.

Reply:
The linked issue is only about updating the defaults for creating new apps, not changing existing apps.

Is there an open issue to change the behavior for deploying existing apps? If not, @fiveisprime can you create one and @neelip can we do that as a separate PR? I have some concerns with that and I think GitHub would be the best place to have that discussion.

Reply:
I believe we discussed that we only want to enable the setting on web app creation rather than on every deploy.

If users already have a working project, we don't want to risk breaking their existing projects by changing an app setting when they deploy.

Reply:
Correct. We don't want to make that kind of change to existing apps on deployment, it doesn't feel right to me. I would be okay with prompting if it's not set and asking them if we should set it for them, but never automatically.

Reply:
If we are planning to enable SCM_DO_BUILD_DEPLOY by default with `Create` in this PR without changing existing behavior we won't be able to replace ".deployment" file with appSetting for SCM_DO... @fiveisprime is that ok? 

I think we need to change the existing behavior in order to replace the ".deployment" file with a appSetting for SCM_DO..., which I agree needs a new PR.